src/ucp/wireup/wireup_ep.c

```diff
@@ -363,8 +363,8 @@ ucp_wireup_ep_do_check(ucp_ep_h ucp_ep, uct_ep_h uct_ep,
 
     wiface = ucp_worker_iface(worker, rsc_idx);
     if (wiface->attr.cap.flags & UCT_IFACE_FLAG_EP_CHECK) {
-        return ucp_ep_do_uct_ep_keepalive(ucp_ep, uct_ep, rsc_idx, flags,
-                                          comp);
+        return ucp_ep_do_uct_ep_keepalive(ucp_ep, uct_ep, UCP_NULL_LANE,
```

Comment:
can we make it so during the initial configuration - before CM is connected, and am_lane points to CM - keepalive_lane will also point to cm lane
then, no need to pass anything special here, ucp_ep_do_uct_ep_keepalive could take keepalive_lane

Reply:
re-implemented to get lane/rsc_idx/uct_ep inside `ucp_ep_do_uct_ep_keepalive`